donut/modules/editor/utils/media_wiki_import.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+import argparse
+from donut.pymysql_connection import make_db
+import re
+import xml.etree.ElementTree as ET
+
+
+def import_file(env, filename):
+    # Create reference to DB
+    db = make_db(env)
+    try:
+        db.begin()
+        tree = ET.parse(filename)
+        root = tree.getroot()
+        query = "INSERT INTO webpage_files(title, last_edit_time, content) VALUES( %s, %s, %s)"
+
+        with db.cursor() as cursor:
+            for elem in root:
+                text = ""
+                date = ""
+                title = ""
+                for ele in elem:
+                    if 'title' in ele.tag:
+                        if 'Image:' in ele.text:
+                            break
+                        title = ele.text
+                    if 'revision' in ele.tag:
+                        for i in ele:
+                            if 'timestamp' in i.tag:
+                                date = i.text
+                            if 'text' in i.tag:
+                                text = i.text
+                if title != '' and text != '' and text is not None:
+                    if date != "":
+                        date = date.replace("T", " ")
+                        date = date.replace("Z", "")
+                    title = title.replace(" ", "_")
+                    text = format_text(text)
+                    cursor.execute(query, [title, date, text])
+        db.commit()
+    finally:
+        db.close()
+
+
+def format_text(text):
+    # Headings
+    matches = re.findall(r"\=+[\+0-9a-zA-Z\(\)\'\* \-\,\?\.]+\=+", text)
```

Comment:
Still think you could use inverted character classes to avoid listing all the possible characters, but as long as it works with the current legacy pages, I'm okay